package.json

```diff
@@ -1,15 +1,17 @@
 {
-  "name":       "purescript-jtable",
-  "license":    "MIT",
+  "name": "purescript-jtable",
+  "version": "0.0.1",
```

Comment:
You may want to bump this to 0.1.0 for better compatibility with semantic versioning (otherwise, they'll be no way to do backward-compatible changes).